charts/astronomer/templates/commander/commander-role.yaml

```diff
@@ -29,20 +29,26 @@ metadata:
     release: {{ $.Release.Name }}
     chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
     heritage: {{ $.Release.Service }}
-  {{- if $.Values.global.enableArgoCDAnnotation }}
+
   annotations:
+    {{- if not $.Values.global.features.namespacePools.namespaces.create }}
+    helm.sh/resource-policy: keep
+    {{- end }}
+    {{- if $.Values.global.enableArgoCDAnnotation }}
     argocd.argoproj.io/sync-wave: "-1"
-  {{- end }}
+    {{- end }}
 rules:
 - apiGroups: [""]
   resources: ["configmaps"]
   verbs: ["create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"]
 - apiGroups: [""]
   resources: ["secrets"]
   verbs: ["create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"]
+{{- if not $.Values.global.disableManageClusterScopedResources }}
```

Comment:
This double negative is unfortunate. Is this something we can change before release, or do we have to live with this?

<https://cleankotlin.nl/blog/double-negations>

Reply:
So the variable is about disableManageClusterScopedResources, and it is set to false by default, once this is enabled we do not manage the namespace in commander that is why the double negative.

Reply:
feature flag MANUAL_NAMESPACE_NAMES is activated in combination with  namespace pools that wont use clusterrole and  clusterrolebinding

Reply:
can you assess the impact if we remove these permissions with namespacepools with some deployments

Reply:
We don't need to solve this double negative. It's just a nuance of how we are configuring this. `disableManageClusterScopedResources` already exists in master branch.